r/vignettes/dataset.Rmd

```diff
@@ -56,18 +56,19 @@ you may need to increase R's download timeout from the default of 60 seconds, e.
 
 ```{r, eval = FALSE}
 bucket <- "https://voltrondata-labs-datasets.s3.us-east-2.amazonaws.com"
-for (year in 2009:2019) {
-  if (year == 2019) {
+for (year in 2009) {
+  if (year == 2022) {
```

Comment:
Perhaps a typo here? This loop only cycles through the single value `2009` and so the year will never be `2022`.

Reply:
Oh good catch. It should cycle through all those years.